pkgs/tools/networking/networkmanager_dmenu/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, glib, fetchFromGitHub, networkmanager, python3Packages
+, gobjectIntrospection, dmenu, makeWrapper }:
+
+let inherit (python3Packages) python pygobject3;
+in stdenv.mkDerivation rec {
+  name = "networkmanager_dmenu-${version}";
+  version = "git-20170301";
+
+  src = fetchFromGitHub {
+    owner = "firecat53";
+    repo = "networkmanager-dmenu";
+    rev = "f631f34645cd581eb14cb2f3286fa02dcc60283f";
+    sha256 = "0f3rkyhhyy3ab0bnaasazx33b952gfl3g3b3sg8fl00n90l53d11";
+  };
+
+  buildInputs = [ glib python pygobject3 makeWrapper gobjectIntrospection networkmanager ];
+
+  phases = "unpackPhase installPhase";
```

Comment:
This probably skip more build phases you want. Does `dontBuild = true` also works here?